src/odemis/gui/win/acquisition.py

```diff
@@ -496,7 +524,7 @@ def on_acquisition_done(self, future):
         self._acq_future_connector = None
 
         try:
-            data, exp = future.result(1) # timeout is just for safety
+            data, exp = future.result(2.0)  # timeout is just for safety
```

Comment:
Why 2 instead of 1?! It should really just be for safety, and shouldn't have any effect normally, as the future is supposed to already be finished.